### DISCUSSION TITLE
Ftp

### DISCUSSION THREAD

**Discussion by glideraiscript** [roleoroleo/yi-hack-Allwinner-v2#138] [Q&A]
Hi all and many thanks to @roleoroleo for this hack.

How can i access to the ftp camera server and to the sd from browser o with php or other solution?

I have the Yi camera home 1080p, hacked it's ok, 
in motion events i have all the list of the saved motion video for play or delete
http://192.168.x.xxx:8080/index.html?page=eventsdir

Can you help me, to access to the camera and sd, via ftp or via browser for save the video in a folder on my pc?

I have activate the transfer to ftp, but don't work.

Thank you.

Regards,
Glider2021

**Comment by roleoroleo** [comment]
You can access the cam with these methods:
- ftp: enable ftp in configuration page and login with an ftp client
- web services: read the wiki
- scp: using ssh password

Check again ftp upload because many users use it. It must work.
